microsoft-edge/progressive-web-apps-chromium/how-to/microsoft-store.md

```diff
@@ -116,6 +116,20 @@ Use this feature to measure distinct traffic from your Store-installed PWA.  Bas
 This feature was first introduced in Microsoft Edge version 91, and the DOM API was introduced in Microsoft Edge version 93.
 
 
+<!-- ====================================================================== -->
+## Created a unified experience across multiple domains
+
+A Web App Manifest is tied to a single domain, but some PWAs use locale-specific domains for their customers in specific regions of the globe. When visiting the PWA in a web browser, customers are seamlessly transitioned from the principal domain (e.g., contoso.com) to a locale-specific one (e.g., contoso.co.ke) because the redirect happens during initial load of that website. Customers who install the PWA from Edge would therefor install it from the locale-specific domain and all future launches of that PWA will go directly to that domain instead of the principal one.
+
```

Comment:
```suggestion
A Web App Manifest is tied to a single domain, but some PWAs use locale-specific domains for their customers in specific regions of the globe. When visiting the PWA in a web browser, customers are seamlessly transitioned from the principal domain (for example, contoso.com) to a locale-specific one (for example, contoso.co.ke) because the redirect happens during initial load of that website. Customers who install the PWA from Edge would therefore install it from the locale-specific domain and all future launches of that PWA will go directly to that domain instead of the principal one.

```